src/UID2.Client/UID2Encryption.cs

```diff
@@ -21,12 +21,14 @@ internal static class UID2Encryption
     {
         public const int GCM_AUTHTAG_LENGTH = 16;
         public const int GCM_IV_LENGTH = 12;
+        // Length of a V2 token, the shortest token currently supported
+        public const int MINIMUM_TOKEN_STRING_LENGTH = 180;
         private static char[] BASE64_URL_SPECIAL_CHARS = { '-', '_' };
 
 
         internal static DecryptionResponse Decrypt(string token, KeyContainer keys, DateTime now, string domainName, IdentityScope identityScope, ClientType clientType)
         {
-            if (token.Length < 4)
+            if (token.Length < MINIMUM_TOKEN_STRING_LENGTH)
```

Comment:
How sure are you we can never have a 179 char token :) possible IM here otherwise...

Reply:
Reverted this